cache.go

```diff
@@ -5,17 +5,26 @@ import (
 	"time"
 )
 
+// CachedValue is a type for cache value
+type CachedValue interface{}
+
+// CachedItems is a map for CachedValue
+type CachedItems map[string]CachedValue
+
+// CachedExpiries is a map for Timer
+type CachedExpiries map[string]*time.Timer
```

Comment:
We don't need the `CachedExpiries` since this value will never change, regardless of the type of object the user caches.